docs/requirements-specification.md

```diff
@@ -54,7 +54,18 @@ _This content item details the use case overview of the system vision in its use
 ### UC3: Get capability overview
 
 ### UC4: Identify capability gap
-
+| UC4 | Identify Capability Gaps
+|---|---|
+| Actors | Lead Researcher |
+| Pre-conditions | - The system contains a comprehensive list of current capabilities of the research group members. |
```

Comment:
"Comprehensive" is a vague term - I would just leave it out completely. Instead, I would specify in the pre-conditions that the system needs to contain both a list of capabilities and a list of research groups and a mapping between them (all the data necessary to identify gaps).